packages/plugin-ext/src/main/browser/languages-main.ts

```diff
@@ -65,6 +89,9 @@ export class LanguagesMainImpl implements LanguagesMain {
         this.proxy = rpc.getProxy(MAIN_RPC_CONTEXT.LANGUAGES_EXT);
     }
 
+    // Map of vscode extension id's to the feature to the handle
+    protected extensionIDToExtensionFeatureMap = new Map<string, Map<FeatureEnum, number>>();
```

Comment:
I cannot find where do we read from it? If it is only required for a subclass, please move it to the subclass then.

Reply:
I've moved it to the subclass